dev/com.ibm.websphere.appserver.org.eclipse.persistence-2.7/com.ibm.websphere.appserver.org.eclipse.persistence-2.7.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.persistence-2.7
+singleton=true
+IBM-Process-Types: server, \
+ client
+-bundles=com.ibm.websphere.appserver.thirdparty.eclipselink.2.7; apiJar=false; location:=dev/api/third-party/, \
+  com.ibm.ws.javaee.persistence.2.2; location:=lib/
+kind=beta
+edition=core
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.persistence-1.0))", \
```

Comment:
same comment here about removing `IBM-Provision-Capability` and `IBM-Install-Policy`